### DISCUSSION TITLE
Can HomeSpan support AP mode?

### DISCUSSION THREAD

**Discussion by yrjyrj123** [HomeSpan/HomeSpan#251] [Ideas]
I have a garage, but no Wi-Fi.
I want to use HomeKit to control the door, can HomeSpan work as an AP? 
This allows my phone to connect directly to HomeSpan and control the garage door without an additional Wi-Fi router.

**Comment by HomeSpan** [comment]
Sorry, the HomeSpan AP can only be used for setup, not for operating the device.  I'm not sure if Apple HomeKit supports such a method.

**Reply by yrjyrj123** [reply]
Thanks for your reply.
